.circleci/config.yml

```
@@ -97,6 +98,29 @@ commands:
           command: |
             curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
             sudo apt update && sudo apt install -y libssl1.1
+  make-artifact:
+    parameters:
+      target:
+        description: "Rust target to put in artifact"
+        type: string
+      suffix:
+        description: "Suffix that is on the binary"
+        type: string
+        default: ""
+    steps:
+      - run:
+          name: Make artifact
+          command: |
+            mkdir shuttle
+            mv target/<< parameters.target >>/release/cargo-shuttle<< parameters.suffix >> shuttle/shuttle<< parameters.suffix >>
```

This also needed to be changed for install support
```suggestion
            mv target/<< parameters.target >>/release/cargo-shuttle<< parameters.suffix >> shuttle/cargo-shuttle<< parameters.suffix >>
```

---

Haha, thanks a million for this @tsar-boomba